Python/Tests/Utilities.Python/PythonTestExplorer.cs

```diff
@@ -151,6 +152,7 @@ public PythonTestExplorer(VisualStudioApp app, AutomationElement element, Automa
 
         public void ClearSearchBar() {
             _searchBar.SetValue("");
+            Thread.Sleep(500);
```

Comment:
new sleep.. started seeing errors related to the serachbar not clearing fast enough